.eslintrc

```diff
@@ -42,8 +46,10 @@
     "accessor-pairs": [2, {"setWithoutGet": true, "getWithoutSet": false}],
     "array-bracket-spacing": [2, "never"],
     "array-callback-return": 2,
+    "arrow-parens": [2, "as-needed"],
```

Comment:
Not entirely sold on this rule/value, since it seems like we're removing parens in a lot of places. But I'm happy to leave it up to a vote. Plus, it seems like the rule is auto fixable (via `--fix`), so should be easy to tweak if if the group leans one way or the other.

http://eslint.org/docs/rules/arrow-parens